app/controllers/contests_controller.rb

```diff
@@ -298,7 +310,9 @@ def supervise
             relation.save
```

Comment:
Should we add a similar check for `if !relation.save` in the `:extra_time case` to be consistent with your addition above?

Reply:
I don't mind, though I don't think there are any cases where validation would fail, I just added the check to be extra safe.